src/main/java/org/broadinstitute/hellbender/dev/tools/walkers/bqsr/DF_BaseRecalibrator.java

```diff
@@ -58,7 +66,7 @@
         usageShort = "Generates recalibration table",
         programGroup = ReadProgramGroup.class
 )
-public class DF_BaseRecalibrator extends CommandLineProgram {
+public class DF_BaseRecalibrator extends DataflowCommandLineProgram {
```

Comment:
So, we have `DF_BaseRecalibrator`, `BQSR_Dataflow`, `BQSR_Dataflow_Transform`, and `BaseRecalibratorUprooted`. This is all very confusing! The existing hellbender convention is to name the standalone pipeline for running a tool `ToolNameDataflow`, and the transform that does the actual work of the tool `ToolNameDataflowTransform` (see, eg., `CountBasesDataflow` and `CountBasesDataflowTransform`).

To bring this more into line with the rest of the codebase, could we rename these classes as follows:
`DF_BaseRecalibrator` -> `BaseRecalibratorDataflow`
`BQSR_Dataflow_Transform` -> `BaseRecalibratorTransform`
`BQSR_Dataflow` -> `BaseRecalibratorDataflowUtils` (since this is just a collection of static utility methods)
`BaseRecalibratorUprooted` -> `BaseRecalibratorWorker` (or whatever other name you can think of that makes sense for this one)